swagger.yaml

```diff
@@ -7647,7 +7676,6 @@ definitions:
       logs_info:
         $ref: '#/definitions/logs_state'
         description: The progress of log collection or empty if logs are not applicable
-        x-go-custom-tag: gorm:"type:varchar(2048)"
```

Comment:
This should be removed for host as well, right?

Reply:
Seems so. Yes